ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -765,7 +761,7 @@
 						TestTargetID = 2D02E47A1E0B4A5D006451C7;
 					};
 					888DAA1429B48C8900311304 = {
-						DevelopmentTeam = J22M3M9VYU;
+						DevelopmentTeam = 578K4HCZ2Y;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The Development Team ID has been changed. If this was intentional and the new team ID is correct, then it's fine. However, if this change was not intended, it could potentially lead to issues with app signing and distribution.

```diff
- DevelopmentTeam = J22M3M9VYU;
+ DevelopmentTeam = 578K4HCZ2Y;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->